src/jdk.jfr/share/classes/jdk/jfr/internal/PlatformEventType.java

```diff
@@ -102,11 +102,11 @@ private static int stackTraceOffset(String name, boolean isJDK) {
             if (isExceptionEvent(name)) {
                 return 4;
             }
-            if (isUsingHandler(name)) {
+            if (isUsingConfiguration(name)) {
                 return 3;
             }
         }
-        return 4;
+        return 3;
```

Comment:
Why is the default stack trace offset changed?

Reply:
Previously the stack looked like this:

   UserEvent::commit() {
   UserEventHandler::write(startTime, duration ...);
   EventWriter::puStackTrace();
   JVM::getStackTraceId(4);

Now it looks like this:

   UserEvent::commit() {
   EventWriter::puStackTrace();
   JVM::getStackTraceId(3);
   
The branches isUsingConfiguration() and isExceptionEvent() are not affected because they used the EventHandler directly previously.